lib/modules/deployment/contract_deployer.js

```diff
@@ -63,7 +63,15 @@ class ContractDeployer {
         }, (err, subRealArgs) => {
           nextEachCb(null, subRealArgs);
         });
-      } else {
+      } else if(arg.substr('.eth') === arg.length - 4) {
+        global.EmbarkJS.Names.resolve(arg, (err, address) => {
+          if(err) {
+            return nextEachCb(__('Error resolving the ens domain'));
```

Comment:
It would be even nicer if you added the arg inside the error with `__('my message %s,' arg)`